scripts/analysis.py

```diff
@@ -1644,7 +1755,7 @@ def plot_in_out_flux(cur, facility, influx_bool, title, is_cum=False, is_tot=Fal
         plt.ylim(bottom=0.0)
         plt.show()
 
-    elif is_cum == True and is_tot == True:
+    elif is_cum and is_tot:
```

Comment:
Relying on this can be a little buggy. See the zen of python. 'explicit is better than implicit'

Reply:
Yea it was originally this way but @pep8speaks told me to change it. Which one do you prefer? I personally think the former (`is_cum == True`) is much better.